templates/managed/apps/index.html

```diff
@@ -133,7 +133,7 @@ <h1>Performant open source with Managed Apps</h1>
     </ul>
     <div class="row">
       <div class="col-8">
-        <p><a href="/managed/contact-us" class="p-button">Contact us</a></p>
+        <p><a href="#get-in-touch" class="p-button js-invoke-modal">Contact us</a></p>
```

Comment:
```suggestion
        <p><a href="/managed/contact-us" class="p-button js-invoke-modal">Contact us</a></p>
```
You should keep this as a link to a static page, as it acts a fallback if the user doesn't have JS enabled